multipaz/src/commonMain/kotlin/org/multipaz/mdoc/zkp/ZkDocument.kt

```diff
@@ -0,0 +1,52 @@
+package org.multipaz.mdoc.zkp
+
+import kotlinx.datetime.Instant
+import kotlinx.io.bytestring.ByteString
+import org.multipaz.cbor.DataItem
+import org.multipaz.crypto.X509CertChain
+
+/**
+ * Represents a document that contains a zero-knowledge (ZK) proof.
+ *
+ * @property zkDocumentData The structured data of the document.
+ * @property proof The ZK proof that attests to the integrity and validity of the document data.
+ */
+data class ZkDocument(
+    val zkDocumentData: ZkDocumentData,
+    val proof: ByteString
+) {
```

Comment:
The CDDL in the standard is

```
ZkDocument = {
    "documentData": ZkDocumentDataBytes, 
    "proof": bstr,
    * tstr => RFU
}
```

and I think it's quite important we return those exact bytes. So this should be changed to be `val documentDataBytes: ByteString` where the comment would say "the bytes of the ZkDocumentDataBytes CBOR"